client/my-sites/domains/domain-management/domain-connect/domain-connect-authorize.jsx

```diff
@@ -31,8 +31,11 @@ class DomainConnectAuthorize extends Component {
 		const { providerId, serviceId, params, translate } = this.props;
 		const { domain } = params;
 
-		wpcom
-			.getDnsTemplateRecords( domain, providerId, serviceId, params )
+		wpcom.req
```

Comment:
Noting that the `wpcom` definition still points to `wp.undocumented()`. We can now remove that and call `wp.req` directly.

Reply:
Addressed that in https://github.com/Automattic/wp-calypso/pull/59146/commits/5b3b6c6e9b9e9eead3493f8b6dd0fbb8feec724a.